notebooks/failure-type-classification/background/testgrid_flakiness_detection.ipynb

```diff
@@ -21,17 +21,17 @@
    "execution_count": 1,
```

Comment:
<p>Could you please run this block?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
@SankBad done :+1: